include/SoftKeyMaskComponent.hpp

```diff
@@ -14,11 +14,31 @@
 
 #include "JuceHeader.h"
 
+class SoftKeyMaskDimensions
+{
+public:
+	SoftKeyMaskDimensions() = default;
+
+	int keyHeight = 60;
+	int keyWidth = 60;
+	int keyCount() const;
+	int rowCount = 6;
+	int columnCount = 1;
+	int height = 480;
+	static const std::uint8_t padding = 10;
```

Comment:
According to https://github.com/Open-Agriculture/AgIsoStack-plus-plus/blob/main/CONTRIBUTING.md:

> Constant values `CAPITALIZED_SNAKE`

And also 

> Where possible, declare variables as `constexpr` if their value can be determined at compile time